### PR TITLE
Added DrizzlePac Drizzle WFPC2 notebook

### PR DESCRIPTION
**Pull request by samanthalh**
The DrizzlePac Drizzle WFPC2 notebook is ready to be added to the spacetelescope notebook repository.

### CODE REVIEW

**Comment by samanthalh** [comment]
This notebook and the initialization notebook both have the same build failure where Travis cannot find the files that are downloaded from CRDS in a previous step. Unclear whether this is due to a problem in the crds command or if travis cannot find certain files even if they're created in a previous step.

**Comment by eteq** [comment]
Quick review:

Mostly this one is good, aside from the obvious problem that it's failing (and the possibility that when it runs it may be long-running?). Hopefully the fix for #41 will also address this and we can cross that bridge only if we have to.

1. The one concern I see is the "Simulate an Image of Constant "Blank Sky" Background" section seems to violate some of the "cell discipline" guidelines from the style guide - it's both a lot of code in one cell *and* has a lot of repition that's probably better addressed via a ``for i in range(1,5)`` instead of copying lines.

**Comment by pllim** [comment]
Is WFPC2 still relevant? Do people still go back and drizzle them by hand? Or do they just go to HLA now and download the final products? 🤔

**Comment by mackjenn** [comment]
Before running section 2, the notebook needs to perform similar operations found in the initialization notebook. Specifically:

os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'
os.environ['CRDS_PATH'] = './reference_files'
!crds bestrefs --files ua0605*_c0m.fits --sync-references=1 --update-bestrefs
os.environ['iref'] = './reference_files/references/hst/uref/'

Since there are still unresolved issues with these commands in that initial notebook, it doesn't make sense to add them yet here until a resolution is found.

**Comment by mackjenn** [comment]
@pllim  Drizzled data products in the archive or HLA are only processed with default parameters so we advise GO's to never use them for science. This example goes one step further and shows how to equalize the different sensitivities on the 4 chips so that the user doesn't have to apply 4 separate zeropoints to a single image. It also allows users to combine data from overlapping chips, (e.g. for dithered observations or multiple orientations)

**Comment by catherine-martlin** [comment]
This one has the same issue as the Initialization notebook -- the path to the reference files is missing a `/` and I'm not certain how to get that added in there correctly. Would it be something to add in after `os.environ['iref'] = os.path.abspath('./reference_files/references/hst/wfc3/')` to instead be something along the lines of `os.environ['iref'] = os.path.abspath('./reference_files/references/hst/wfc3/') + '/' ` ?

**Comment by catherine-martlin** [comment]
@eteq - I think that the error for this has now changed. I was able to get the notebook to run/convert successfully, but in the "check notebooks" Travis failed. I tried poking around that but don't really understand what issue it may be having there. Thoughts on that?

**Comment by mackjenn** [comment]
I'm wondering if the updatewcs command is able to run with 'use_db=True'.   (In the initialization notebook, we set it to false.)

The issue with 'True' is documented here and there appears to be a fix, but not sure in which version
https://github.com/spacetelescope/stwcs/issues/70#issuecomment-450999976

**Comment by eteq** [comment]
Aha, it seems the error was do to a transient network problem.  It seems to work now after I manually re-ran it...

File: notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'\n",
+    "os.environ['CRDS_PATH'] = os.path.abspath('./reference_files')\n",
+    "\n",
+    "subprocess.check_output('crds bestrefs --files ua0605*_c0m.fits --sync-references=1 --update-bestrefs', shell=True)\n",
+    "\n",
+    "os.environ['uref'] = str(os.path.abspath('./reference_files/references/hst/wfpc2/')) + '/'"
```
[the remark this answers is not in the document]
Comment (tddesjardins) [reply notebooks/DrizzlePac/drizzle_wfpc2/drizzle_wfpc2.ipynb]: Updated to `%matplotlib inline` and removed the `plt.show()` commands. Added the `os.path.join()` commands.

**Pull request merged**